src/components/Common/TooltipTimestamp.tsx

```diff
@@ -33,7 +35,9 @@ const TooltipTimestamp = ({
       title={formatTooltipTimestamp(timestamp)}
     >
       <Typography variant={variant} sx={styles.tooltipText}>
-        {formatTimestamp(timestamp)}
+        {isEdited
+          ? formatTimestamp(timestamp, true)
+          : formatTimestamp(timestamp)}
```

Comment:
This can be simplified:
```suggestion
        {formatTimestamp(timestamp, isEdited)}
```

Reply:
Done